modules/Movecraft/src/main/java/net/countercraft/movecraft/craft/BaseCraft.java

```diff
@@ -598,4 +601,17 @@ public void setTotalFuel(double fuel) {
     public double getTotalFuel() {
         return totalFuel;
     }
+
+    @Override
+    public Map<NamespacedKey, TrackedLocation> getTrackedLocations() {return trackedLocations;}
```

Comment:
Each `NamespacedKey` should point to a `Set<TrackedLocation>`).  This allows each add-on to have multiple tracked locations under one key.